_projects/2022/100385586/100385586.Rmd

```diff
@@ -224,11 +272,13 @@ Lastly, we have to include the most important characteristics of the graph. Whil
 
 This evolutionary feature is provided by GGanimated. Notice how each country change the colour depending on the position within the graph. 
 
-```{r, fig.align='center', fig.height = 5, fig.width = 9.5, preview=TRUE}
-p + transition_states(year) 
+```{r, fig.align='center', fig.height = 5, fig.width = 9.5}
```

Comment:
You removed the `preview=TRUE` option. Please bring it back unless you are using it in another chunk.